### ISSUE TITLE
Project sync mixes up contexts due to race condition

### ISSUE THREAD

**Issue by vbradnitski** [enonic/xp#8555]
nodeService.refresh() command is missing, so there is a time gap between new content data update and it's accessibility from search.
